content/en/docs/tasks/configmap-secret/managing-secret-using-config-file.md

```diff
@@ -170,7 +170,56 @@ type: Opaque
 
 `YWRtaW5pc3RyYXRvcg==` decodes to `administrator`.
 
-## Clean up
+## Edit a Secret {#edit-secret}
+
+To edit the data in the Secret you created using a configuration
+file, modify the `data` or `stringData` field in your original configuration
+file and apply the file to your cluster. You can edit an existing `Secret` object unless it is [immutable](/docs/concepts/configuration/secret/#secret-immutable).
+
+For example, if you want to change the password from the previous example to
+`birdsarentreal`, do the following:
+
+1. Encode the new password string:
+
+   ```shell
+   echo -n 'birdsarentreal' | base64
+   ```
+
+   The output is similar to:
+
+   ```
+   YmlyZHNhcmVudHJlYWw=
+   ```
+
+1. Update the `data` field with your new password string:
+
+   ```yaml
+   apiVersion: v1
+   kind: Secret
+   metadata:
+     name: mysecret
+   type: Opaque
+   data:
+     username: YWRtaW4=
+     password: YmlyZHNhcmVudHJlYWw=
+   ```
+
+1. Apply the file to your cluster:
+
+   ```shell
+   kubectl apply -f ./secret.yaml
+   ```
+
+   The output is similar to:
+
+   ```
+   secret/mysecret configured
+   ```
+
+Kubernetes updates the existing `Secret` object, because you supplied a
+configuration file that had updated data with the same object name.
+
+## Delete a Secret {#clean-up}
```

Comment:
```suggestion
## Clean up
```

We have a convention for what this heading is named (in fact, as a separate concern, we could consider adding a localized string for it.

Reply:
So the reason I renamed it was because I saw it as a subtask, deleting a secret in a cluster, as opposed to cleaning up the resources created on this task page. I agree that the heading should normally be cleanup though

Reply:
(done in the latest push)